multiversx_sdk_cli/delegation/staking_provider.py

```diff
@@ -171,11 +171,13 @@ def prepare_transaction_for_unjailing_nodes(self, owner: IAccount, args: Any) ->
         delegation_contract = Address.new_from_bech32(args.delegation_contract)
 
         public_keys = self._load_validators_public_keys(args)
+        amount = int(args.value)
 
         tx = self._factory.create_transaction_for_unjailing_nodes(
             sender=owner.address,
             delegation_contract=delegation_contract,
-            public_keys=public_keys
+            public_keys=public_keys,
+            amount=amount
```

Comment:
Should have been a separate PR from the one that alters `sc-meta` installation flow.

Reply:
I'll move the alteration of sc-meta in another PR.